src/App.vue

```diff
@@ -1,10 +1,52 @@
 <script setup lang="ts">
-// This starter template is using Vue 3 <script setup> SFCs
-// Check out https://v3.vuejs.org/api/sfc-script-setup.html#sfc-script-setup
 import NewLexemeForm from '@/components/NewLexemeForm.vue';
+import { Message as WikitMessage } from '@wmde/wikit-vue-components';
 import '@wmde/wikit-vue-components/dist/wikit-vue-components.css';
 </script>
 
 <template>
-	<new-lexeme-form />
+	<p class="wbl-snl-intro-text">
+		To Be Done
+	</p>
+	<div class="wbl-snl-app">
+		<new-lexeme-form />
+		<wikit-message type="notice">
+			To Be Done
+		</wikit-message>
+	</div>
 </template>
+
+<style lang="scss" scoped>
+@import "@wmde/wikit-tokens/variables";
+@import "@wmde/wikit-vue-components/src/styles/mixins/Typography";
+
+$break-point: 981px;
+
+.wbl-snl-intro-text {
+	@include body;
+}
+
+.wbl-snl-app {
+	display: flex;
+
+	@media (min-width: $break-point) {
+		align-items: stretch;
+
+		& > * + * {
+			margin-inline-start: $dimension-layout-medium;
+		}
+	}
+
+	@media (max-width: $break-point) {
```

Comment:
Doesn’t one of these two `@media` queries need to be offset by one? Otherwise at exactly 981px width, both will apply.

Reply:
Good point, Figma says `981px and below`, so the other one get's a `+1`

Reply:
Now it should work